src/encoded/tests/test_search.py

```diff
@@ -596,7 +671,49 @@ def test_index_data_workbook(app, workbook, testapp, indexer_testapp, htmltestap
                 html_res = htmltestapp.get(item_res['@id'])
                 assert html_res.body.startswith(b'<!DOCTYPE html>')
             except Exception as e:
+                ignored(e)
                 pass
+    if ItemTypeChecker.CONSIDER_DELETED:
+        print(f"(CONSIDER_DELETED) Items deleted = {ItemTypeChecker.DELETED_SEEN}")
+        print(f"db_total={db_total} es_total={es_total} deleted_seen={ItemTypeChecker.deleted_seen()}")
+        assert(db_total == es_total + ItemTypeChecker.deleted_seen())  # 2nd is db, 4th is es
+    else:
+        print(f"(not CONSIDER_DELETED) db_total={db_total} es_total={es_total} deleted_seen={ItemTypeChecker.deleted_seen()}")
+        assert(db_total == es_total)  # 2nd is db, 4th is es
+
+@pytest.mark.manual
+@pytest.mark.skip
+def test_index_data_workbook_after_posting_deleted_page_c4_570(workbook, testapp, html_testapp):
```

Comment:
This test has to be invoked manually. It takes a very long time to run, but is potentially useful for debugging. It was imported from `cgap-portal`, where the bug it is a regression test for was found.